api/pkg/s3/utils.go

```diff
@@ -141,3 +144,23 @@ func setXmlHeader(w http.ResponseWriter, body []byte) {
 /*func hasServerSideEncryptionHeader(header http.Header) bool {
 	return crypto.S3.IsRequested(header) || crypto.SSEC.IsRequested(header)
 }*/
+
+func WriteS3ErrorResponse(response *restful.Response, err error) {
+	var httpStatus int
+	s3ErrorCode, ok := err.(s3error.S3Error)
+	if ok {
```

Comment:
In "api/pkg/s3/api-response.go" file, there are some methods to create response.  I think the two methods should be placed at file "api/pkg/s3/api-response.go" and merge the methods that has the same function